pkg/sql/logictest/testdata/logic_test/geospatial

```diff
@@ -249,25 +251,27 @@ SELECT
   ST_AsEWKB(geog),
   ST_AsHexWKB(geog),
   ST_AsHexEWKB(geog),
```

Comment:
this output is getting a little harder to read. do you mind moving HexWKB/HexEWKB to a new query?

Reply:
Done.